### PR TITLE
SWC-5623: set container that should be used for tooltip to select the…

### PR DESCRIPTION
**Pull request by jay-hodgson**
… body element

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Initially I thought it would be great to reuse @nickgros 's EntityBadgeIcons from SRC.  Plan to save that for the next step, when files browser is replaced by SRC.

File: src/main/java/org/sagebionetworks/web/client/DisplayUtils.java [hunk 1]
```diff
@@ -622,6 +622,7 @@ public static Tooltip addTooltip(Widget widget, String tooltipText, Placement po
 		t.setIsHtml(true);
 		t.setIsAnimated(false);
 		t.setTrigger(Trigger.HOVER);
+		t.setContainer("body");
```
Comment (nickgros) [inline src/main/java/org/sagebionetworks/web/client/DisplayUtils.java:625]: This is some Bootstrap magic, right?

**Review by nickgros — APPROVED** [review]

File: src/main/java/org/sagebionetworks/web/client/DisplayUtils.java (hunk 1 above)
Comment (jay-hodgson) [reply src/main/java/org/sagebionetworks/web/client/DisplayUtils.java:625]: Yes

Comment (jay-hodgson) [reply src/main/java/org/sagebionetworks/web/client/DisplayUtils.java:625]: I think the default behavior is to attach the new (absolute positioned) tooltip element as a sibling to the target element, but this allows you to specify another element.

**Pull request merged**